jdi-light/src/main/java/com/epam/jdi/light/asserts/core/DataListAssert.java

```diff
@@ -145,7 +144,7 @@ private boolean isDisplayed(T element) {
      */
     @JDIAction("Assert that '{name}' is empty")
     public DataListAssert<T, D> empty() {
-        jdiAssert(element.isEmpty() ? "list is empty" : "list is not empty", Matchers.is("list is empty"));
+        jdiAssert(element().isEmpty() ? "list is empty" : "list is not empty", Matchers.is("list is empty"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5576697)